pkg/runtime/hotreload/loader/disk/disk.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package disk
```

Comment:
Could you share more thoughts about disk and operator?

Reply:
yes please @JoshVanL I think the distinction could use some explanation via comments. I believe a `README.md` (at least) in the `hotreload` folder would be beneficial. Maybe even in the subfolders (differ/loader/reconciler) just to be explicit